### PR TITLE
update : grid

### PR DESCRIPTION
**Pull request by Bibin-Antony**

### CODE REVIEW

**Comment by Bibin-Antony** [comment]
image

**Review by hiteshverma27 — COMMENTED** [review]
Looks good

**Review by biswanath-tewari — APPROVED** [review]

File: Container/CSS/grid.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    grid-template-columns: repeat(2, 14rem);
+    justify-content: center;
+    padding: 2rem 0rem;
+}
+
+.twoRow-grid div,.threeRow-grid div{
+    outline: 1px solid black;
+    min-width: 4rem;
+    height: 9rem;
+    background-color: rgb(120, 120, 170);
```
Comment (Naveenchandar) [inline Container/CSS/grid.css]: Please use hex codes for consistency.

File: Container/CSS/grid.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+.threeRow-grid{
+    display: grid;
+    background-color: lightpink;
```
Comment (Naveenchandar) [inline Container/CSS/grid.css]: Please use hex codes for consistency.

File: Container/CSS/grid.css (hunk 1 above)
Comment (Bibin-Antony) [reply Container/CSS/grid.css]: ok will do.

File: Container/CSS/grid.css (hunk 2 above)
Comment (Bibin-Antony) [reply Container/CSS/grid.css]: will do.

**Review by ShoaibTamboli — COMMENTED** [review]
looks good to me

**Pull request merged**